packages/containers/src/ShortcutManager/ShortcutManager.container.js

```diff
@@ -0,0 +1,71 @@
+import React, { PropTypes } from 'react';
+import keycode from 'keycode';
+import keys from 'lodash/keys';
+import { componentState } from 'react-cmf';
+
+/**
+ * ShortcutManager matches shortcuts to the current route to redirect to a new
+ * one
+ *
+ * @extends {React}
+ */
+class ShortcutManager extends React.Component {
+	static displayName = 'Container(ShortcutManager)';
+	static propTypes = {
+		...componentState.propTypes,
+	};
+
+	static contextTypes = {
+		router: PropTypes.object.isRequired,
+	};
+
+	constructor(props) {
+		super(props);
+		this.redirect = this.redirect.bind(this);
+		this.handleKeyPress = this.handleKeyPress.bind(this);
+		this.handleRegexMatching = this.handleRegexMatching.bind(this);
+	}
+
+	componentDidMount() {
+		document.addEventListener('keydown', this.handleKeyPress);
+	}
+
+	// eslint-disable-next-line class-methods-use-this
+	componentWillUnmount() {
+		document.removeEventListener('keydown');
+	}
```

Comment:
if this is possible (to mock document) I would like to see unittest on didMount and WillUnmout.